src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double katetA;
+    private final double katetB;
+
+    public RightTriangle(double katetA, double katetB) {
+        this.katetA = katetA;
+        this.katetB = katetB;
+        setColor(new ColorSupplier().getRandomColor());
```

Comment:
You should not generate random values inside the figure classes. Let's generate random values in `FigureSupplier` and pass them to the constructor of your figures.